packages/runtime/src/widgets/DataGrid/DataGrid.tsx

```diff
@@ -64,16 +67,32 @@ export interface DataGridRowTemplate {
 }
 
 export const DataGrid: React.FC<GridProps> = (props) => {
+  const [selectedRow, setSelectedRow] = useState<object[]>();
```

Comment:
Plural selectedRows

Reply:
since its already being used below in rowSelection's onChange so i am renaming to rowsSelected